processor/src/chiplets/aux_trace/mod.rs

```diff
@@ -159,24 +84,24 @@ impl<E: FieldElement<BaseField = Felt>> AuxColumnBuilder<E> for ChipletsVTableCo
 
     fn get_responses_at(&self, main_trace: &MainTrace, alphas: &[E], row: usize) -> E {
         chiplets_vtable_add_sibling(main_trace, alphas, row)
-            * chiplets_kernel_table_include(main_trace, alphas, row)
+            * build_kernel_procedure_table_inclusions(main_trace, alphas, row)
     }
 }
 
-// CHIPLETS VIRTUAL TABLE REQUESTS
+// VIRTUAL TABLE REQUESTS
 // ================================================================================================
 
-/// Constructs the inclusions to the table when the hasher absorbs a new sibling node while
-/// computing the old Merkle root.
-fn chiplets_vtable_add_sibling<E>(main_trace: &MainTrace, alphas: &[E], row: usize) -> E
+/// Constructs the removals from the table when the hasher absorbs a new sibling node while
+/// computing the new Merkle root.
+fn chiplets_vtable_remove_sibling<E>(main_trace: &MainTrace, alphas: &[E], row: usize) -> E
 where
     E: FieldElement<BaseField = Felt>,
 {
-    let f_mv: bool = main_trace.f_mv(row);
-    let f_mva: bool = if row == 0 { false } else { main_trace.f_mva(row - 1) };
+    let f_mu: bool = main_trace.f_mu(row);
+    let f_mua: bool = if row == 0 { false } else { main_trace.f_mua(row - 1) };
```

Comment:
With `row - 1`, we "look back". However, during constraint evaluation, we receive an `EvaluationFrame`, which only "looks forward" (i.e. you get the rows at `i` and `i+1`). Is this not going to cause a problem?

Would it not be clearer to generate the auxiliary column by only referencing `row` and `row + 1`?

Reply:
The use of `row - 1` is only due to me not been able to come up with a better solution that gives meaning to "...on rows which are 1 less than a multiple of 8" for the edge case of `row == 0`.